test/FieldTests/PredictableNumberGenerator.java

```diff
@@ -0,0 +1,44 @@
+package FieldTests;
+
+import Field.NumberGenerator;
+
+public class PredictableNumberGenerator implements NumberGenerator {
+
+    private final int numberOfMines;
+    private final int mine1Row;
+    private final int mine1Column;
+    private final int mine2Row;
+    private final int mine2Column;
+    private int calledTimes = 0;
+
+    // public PredictableNumberGenerator(List<Coordinates> mines) { }
```

Comment:
Commented out code

Reply:
I have pushed up the latest of this branch which should have no comments!